docs/troubleshooting/kubernetes.rst

```diff
@@ -25,3 +235,226 @@ If you've seen this error, you can override or change the default traffic-group
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Ping the Node's VTEP IP address.
+
+   Use the ``-s`` flag to set the MTU of the packets to allow for VxLAN encapsulation. ::
+
+     ping -s 1600 <OSE_Node_IP>
+
+#. Increase the log level to DEBUG and monitor the |kctlr| log files.
+
+   - Get the name of the k8s-bigip-ctlr for which you want to view logs. Be sure to specify the namespace the |kctlr| runs in. ::
+
+       oc get pod --namespace=kube-system
+       NAME                             READY     STATUS    RESTARTS   AGE
+       k8s-bigip-ctlr-687734628-7fdds   1/1       Running   0          15d
+
+   - Add the log-level=DEBUG annotation to the |kctlr| Deployment YAML file. ::
+
+       oc annotate myDeployment.yaml "--log-level=DEBUG" --namespace=kube-system
+
+   - Follow the logs. ::
+
+       oc logs -f k8s-bigip-ctlr-687734628-7fdds --namespace=kube-system
+
+#. Output the REST requests from the BIG-IP logs.
+
+   - Using the BIG-IP CLI, do a ``tcpdump`` of the underlay network. ::
+
+       tcpdump -i <name-of-BIG-IP-VXLAN-tunnel>
+
+     \
+
+     .. code-block:: console
+        :caption: Example showing two-way communication on port 4789 between the BIG-IP VTEP IP and the OSE node VTEP IPs.
+
+        tcpdump -i ocpvlan
+        08:08:06.933951 IP 10.214.1.102.58472 > 10.214.1.23.4789: VXLAN, flags [I] (0x08), vni 0
+        IP 10.130.0.27.http > 10.128.2.10.37542: Flags [.], ack 9, win 219, options [nop,nop,TS val 573988389 ecr 3961177660], length 0 in slot1/tmm1 lis=_wcard_tunnel_/Common/ose-tunnel
+        08:08:06.934310 IP 10.214.1.23.28277 > 10.214.1.102.4789: VXLAN, flags [I] (0x08), vni 0
+        IP 10.128.2.10.37542 > 10.130.0.27.http: Flags [.], ack 923, win 251, options [nop,nop,TS val 3961177661 ecr 573988389], length 0 out slot1/tmm0 lis=_wcard_tunnel_/Common/ose-tunnel
+
+   - Using the BIG-IP CLI, do a ``tcpdump`` of the overlay network. ::
+
+       tcpdump -i <name-of-BIG-IP-VXLAN-tunnel>
+
+     \
+
+     .. code-block:: console
+        :caption: Example showing traffic on the overlay network; at minimum, you should see BIG-IP health monitors for the pod IP addresses.
+
+         tcpdump -i ose-tunnel
+         08:09:51.911667 IP 10.128.2.10.38036 > 10.130.0.27.http: Flags [.], ack 1, win 229, options [nop,nop,TS val 3961282640 ecr 574093366], length 0 out slot1/tmm0 lis=
+         08:09:51.911672 IP 10.128.2.10.38036 > 10.130.0.27.http: Flags [P.], seq 1:8, ack 1, win 229, options [nop,nop,TS val 3961282640 ecr 574093366], length 7 out slot1/tmm0 lis=
+         08:09:51.913161 IP 10.130.0.27.http > 10.128.2.10.38036: Flags [.], ack 8, win 219, options [nop,nop,TS val 574093369 ecr 3961282640], length 0 in slot1/tmm0 lis=
+         08:09:51.913265 IP 10.130.0.27.http > 10.128.2.10.38036: Flags [P.], seq 1:922, ack 8, win 219, options [nop,nop,TS val 574093369 ecr 3961282640], length 921 in slot1/tmm0 lis=
+
+#. Use the BIG-IP CLI to view VLAN statistics.
```

Comment:
Change to "Using the BIG-IP CLI, view statistics of the underlay and overlay networks."